packages/web-app/src/components/depositInput/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import styled from 'styled-components';
+import { ButtonText } from '@aragon/ui-components';
+import { StyledInput } from '../../../../ui-components/src/components/input/textInput';
+
+export type DepositInputProps =
+    React.InputHTMLAttributes<HTMLInputElement> & {
+        /** Text that appears on the button present on the right side of the input; if no text
+         * is provided, the button will not be rendered */
+        adornmentText?: string;
+        /** Handler for when the button present on the right side of the input  is
+         * clicked */
+        onAdornmentClick?: () => void;
+        /** Changes a input's color schema */
+        mode?: 'default' | 'success' | 'warning' | 'critical';
+        /** Disable the input but keep the adornment button active */
+        disabledFilled?: boolean;
+    };
+
+export const DepositInput = React.forwardRef<
```

Comment:
There is not any component like this already created? It is no possible to use the previous component?